web/packages/teleport/src/Discover/Database/SetupAccess/SetupAccess.tsx

```diff
@@ -207,15 +207,7 @@ function DbEngineInstructions({
         return (
           <Box mb={3}>
             <Text mb={2}>
-              Database users must allow{' '}
-              <Link
-                href="https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/UsingWithRDS.IAMDBAuth.DBAccounts.html#UsingWithRDS.IAMDBAuth.DBAccounts.PostgreSQL"
-                target="_blank"
-              >
-                IAM authentication
-              </Link>{' '}
-              in order to be used with Database Access for RDS. To enable, users
-              must have a <Mark>rds_iam</Mark> role:
+              Users must have a <Mark>rds_iam</Mark> role:
```

Comment:
I know this said "a" rds_iam role previously, and "a" is correct if we removed the role name and just said "a role". But it reads weird and if this will always be `rds_iam` here, I think we should change to `Users must have an rds_iam role"`. However, feel free to skip this as i can see it working both ways